src/client/camera.cpp

```diff
@@ -45,6 +45,140 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define WIELDMESH_AMPLITUDE_X 7.0f
 #define WIELDMESH_AMPLITUDE_Y 10.0f
 
+// Returns the fractional part of x
+inline f32 my_modf(f32 x)
+{
+	double dummy;
+	return modf(x, &dummy);
+}
```

Comment:
```suggestion
inline f32 my_modf(f32 x)
{
	f32 dummy;
	return modff(x, &dummy);
}
```
(I know you just copied it but this can be improved)